src/components/picker/helpers/usePickerSelection.tsx

```diff
@@ -1,12 +1,21 @@
 import {RefObject, useCallback, useState, useEffect} from 'react';
 import _ from 'lodash';
-import {PickerProps, PickerValue, PickerSingleValue, PickerMultiValue, PickerModes} from '../types';
+import {
+  PickerProps,
+  PickerValue,
+  PickerSingleValue,
+  PickerMultiValue,
+  PickerModes,
+  PickerPropsDeprecation
+} from '../types';
+import {ModalTopBarProps} from '../../modal/TopBar';
 
-interface UsePickerSelectionProps
-  extends Pick<PickerProps, 'migrate' | 'value' | 'onChange' | 'getItemValue' | 'topBarProps' | 'mode'> {
-  pickerExpandableRef: RefObject<any>;
-  setSearchValue: (searchValue: string) => void;
-}
+type UsePickerSelectionProps = Pick<PickerProps, 'value' | 'onChange' | 'mode'> &
+  Pick<PickerPropsDeprecation, 'migrate' | 'getItemValue' | 'topBarProps'> & {
+    headerProps?: ModalTopBarProps;
```

Comment:
Why do you need the headerProps here?

Reply:
It's redundant removed, I forgot it.

Reply:
Can't it be of type DialogHeaderProps?

Reply:
No it can't be `DialogHeaderProps` since the `onCancel` is `ModalTopBarProps` type

Reply:
done in the new `onCancel` commit